docs/stark-build/NG_CLI_BUILD_CUSTOMIZATIONS.md

```diff
@@ -117,9 +117,17 @@ Webpack plugins as well as some utils in order to leverage some functionality to
 
 This is the list of utils and plugins used by the Stark-Build package:
 
+### Loaders
+
+### [tslint-loader](https://github.com/wbuchwalter/tslint-loader)
+
+This loader [lints](https://palantir.github.io/tslint/) your TypeScript files based on the configuration in your `tslint.json`.
+
+_Because at this moment disabling `typeCheck` in this loader drasticly improves build time we opted to overwrite and disable all [tslint rules](https://palantir.github.io/tslint/rules/) that require Type Info._
```

Comment:
Maybe you can put this in bold letters so it really stands out

Reply:
I was treating it more like a footnote, because I thought the text above it is more important. But I updated it. 😄